appinventor/appengine/src/com/google/appinventor/server/storage/ObjectifyStorageIo.java

```diff
@@ -2040,6 +2048,13 @@ public void run(Objectify datastore) throws IOException {
                 // Only include screenshots if asked...
                 continue;
               }
+              if (fileName.endsWith("scm") || fileName.endsWith("blk") || fileName.endsWith("bky") || (fileName.endsWith("yail") && includeYail)) {
```

Comment:
We can't compile App Inventor v1 blocks any more, so we shouldn't include "blk" as part of this check.